src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -229,6 +229,24 @@
   {{ block('button_widget') }}
 {%- endblock reset_widget -%}
 
+{%- block clickable_avatar_widget -%}
+  {%- set classes = (attr.class|default('') ~ ' img-thumbnail clickable-avatar')|trim -%}
+
+  <a href="{{ click_url }}" target="_blank">
+    <img class="{{ classes }}" src="{{ value }}" alt="{{ image_alt }}">
+  </a>
+{%- endblock clickable_avatar_widget -%}
+
+{%- block addons_connect_widget -%}
```

Comment:
Same here: if we dont reuse it, please do not make widgets as I'm worried about premature abstraction